dkist/tests/generate_eit_test_dataset.py

```diff
@@ -7,12 +7,13 @@
 from astropy.io import fits
 from astropy.modeling.models import (AffineTransformation2D, Multiply,
                                      Pix2Sky_TAN, RotateNative2Celestial, Shift)
+from astropy.table import Table
 from astropy.time import Time
 from gwcs import coordinate_frames as cf
 from sunpy.time import parse_time
 
 from dkist_inventory.asdf_generator import references_from_filenames
-from dkist_inventory.inventory import table_from_headers
+from dkist_inventory.header_parsing import HeaderParser
```

Comment:
Strictly speaking, I'd prefer to see changes needed for the new version of `dkist-inventory` in their own separate PR. It's not a hill I'm going to die on, though.

Reply:
Both these files are just to generate the test data when we update the asdf schemas, so I have no issues about them being hacky and tied to whatever version of inventory I like :grinning: